src/os/bluestore/BlueStore.h

```diff
@@ -1899,10 +1899,12 @@ class BlueStore : public ObjectStore,
   uint64_t osd_memory_cache_min = 0; ///< Min memory to assign when autotuning cache
   double osd_memory_cache_resize_interval = 0; ///< Time to wait between cache resizing 
   double max_defer_interval = 0; ///< Time to wait between last deferred submit
+  uint32_t config_changed = 0; ///< Counter to determine if there is a configuration change. Protected by config_lock.
```

Comment:
You might want to use atomic instead which avoids the need for the additional lock

Reply:
I have used atomic now. Please verify.